docs/Pantheon-API/RPC-PubSub.md

```diff
@@ -207,16 +207,16 @@ The logs subscription returns [log objects](../Reference/Pantheon-API-Objects.md
 Use the `newPendingTransactions` parameter with `eth_subscribe` to be notified of pending transactions 
 added to the transaction pool for the node. 
 
-The pending transactions subscription returns the transaction hashes of the pending transactions. 
+The pending transactions subscription returns the transaction hashes or transaction details of the pending transactions. 
 
 If a chain reorganization occurs, transactions are resubmitted to be included in the new canonical chain. 
 This means the subscription can publish notifications for the same pending transaction more than once.
 
-!!!example
-    To subscribe to pending transaction notifications:
+!!!example "Transaction Hashes"
+    To subscribe to pending transaction notifications and receive transaction hashes only:
     
     ```json
-    {"id": 1, "method": "eth_subscribe", "params": ["newPendingTransactions"]}
+    {"id": 1, "method": "eth_subscribe", "params": ["newPendingTransactions", {"includeTransactions":false}]}
```

Comment:
Do we have to provide this parameter and if not, what's the default value ?

Reply:
Default is transaction hashes only.  Updated to indicate this.